atef/bin/config.py

```diff
@@ -29,8 +63,8 @@ def build_arg_parser(argparser=None):
     return argparser
 
 
-def main(filenames: Optional[List[AnyPath]] = None):
-    app = QApplication([])
+def main(filenames: Optional[List[AnyPath]] = None, **kwargs):
+    app = QApplication(sys.argv)
```

Comment:
I'm having a bit of a moment trying to remember how these subparsers work. It looks like **kwargs was added here, then not used, and then the entirety of the cli invocation is passed into QApplication? What exactly is going on? How does QApplication respond if other non-qt-args get into sys.argv?

Reply:
Emperically, it looks like kwargs has all the arg values, so it has to be included just to be called, and QApplication seems to run just fine even if you give it extra arguments. Weird stuff.